atlasdb-commons/src/main/java/com/palantir/common/base/BatchingVisitableView.java

```diff
@@ -191,8 +191,8 @@ public T getMin(final Ordering<? super T> o, @Nullable T defaultElement) {
     }
 
     /**
-     * Returns an immutable copy of the current contents of this iterable view. Does not support
-     * null elements.
+     * Returns an immutable List of all the objects in the iterable view (not a single batch).
+     * Does not support null elements.
```

Comment:
Since we're cleaning up the javadoc, maybe also replace this with "@throws NullPointerException if any of the objects is null"?  (To me, "Does not support" is unclear)